Makefile

```diff
@@ -51,7 +51,12 @@ check:
 #
 # Example:
 #   make verify
+ifeq ($(SKIP_BUILD), true)
+$(info build is being skipped)
```

Comment:
Why do you print the message is what I'm asking.  The message is irrelevant
- not worth the extra lines in the makefile.

On Tue, Sep 29, 2015 at 3:51 PM, David Eads <notifications@github.com>
wrote:

> In Makefile
> <https://github.com/openshift/origin/pull/4825#discussion_r40672946>:
>
> > @@ -51,7 +51,12 @@ check:
> >  #
> >  # Example:
> >  #   make verify
> > +ifeq ($(SKIP_BUILD), true)
> > +$(info build is being skipped)
>
> Why?
>
> This was the intent of SKIP_BUILD all along, but the dependency on verify
> when called transitively through test resulted in a build regardless of
> intent
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4825/files#r40672946>.
>